armbian/base/scripts/bbb-cmd.sh

```diff
@@ -16,9 +16,9 @@ possible commands:
   base restart      restarts the node
   base shutdown     shuts down the node
   
-  usb_thumbdrive    <check|mount|umount>
-  usb_backup
-
+  usb_flashdrive    <check|mount|umount>
+  backup            <create|restore|hsm_secret_create|hsm_secret_restore>
```

Comment:
nit: The naming here could be confusing: `bbb-cmd.sh backup hsm_secret_create`. 

> Does it create a new `hsm_secret` or does it create a new backup of the `hsm_secret` ??

Reading through the code makes it clear and I know the latter is meant, but will that be obvious to somebody else starting to work with it?

Maybe split it up into a `backup` and `restore` module? like e.g.
```
bbb-cmd.sh backup hsm_secret
bbb-cmd.sh restore hsm_secret
```

Your call.

Reply:
How about this?
```
bbb-cmd.sh backup sysconfig
bbb-cmd.sh backup hsm_secret
bbb-cmd.sh restore sysconfig
bbb-cmd.sh restore hsm_secret
```

Reply:
Sounds good!

Reply:
Done.